docs/redoc/master/openapi.json

```diff
@@ -6585,7 +6585,8 @@
             "type": "array",
             "items": {
               "type": "string"
-            }
+            },
+            "uniqueItems": true
```

Comment:
This might lead to a breaking change in clients, is it possible to remove?

Reply:
I don't think client enforces this one. I propose to ignore for now